java/test/jmri/jmrit/logixng/util/parser/functions/ConvertFunctionsTest.java

```diff
@@ -93,17 +92,28 @@ public void testIsIntFunction() throws Exception {
         
         Assert.assertFalse((boolean)isIntFunction.calculate(symbolTable, getParameterList(expr_boolean_true)));
         Assert.assertFalse((boolean)isIntFunction.calculate(symbolTable, getParameterList(expr_str_HELLO)));
+        JUnitAppender.suppressWarnMessageStartsWith("the string \"hello\" cannot be converted to a number");
```

Comment:
Thanks for doing this!

Is there a reason for suppress instead of assert? If the message is asserted, a developer will need to think twice before removing it, which is good. In this class, I would prefer assert instead of suppress since the warnings tell the user he has done something wrong.

Reply:
I wan't 100% sure if this an actual Warn, or if user is informed via other means, ia via UI.
No problems with changin this to an Assert ;-)